releasenotes.md

```diff
@@ -4,9 +4,9 @@
 
 * Added a parallel cluster example that explains how to use AWS ParallelCluster to build HPC compute cluster using trn1 compute nodes to run the distributed ML training job.
 
-# Known Issues
+## Known Issues
 
-* **Name or service not known** 
+### **Name or service not known**
```

Comment:
i think with latest pcluster (3.3 and above)  this should be fixed ?

Reply:
Fixed. See new commit: https://github.com/aws-neuron/aws-neuron-parallelcluster-samples/pull/17/commits/6456e8eeec7c015832c27d3f05e3d197ca642249